index.html

```diff
@@ -37,9 +43,235 @@ <h1>Donice Ubaru</h1>
       <button type="button">Start Collaboration</button>
     </div>
 
-    <img src="./images/geometry.svg" id="geometry-image" alt="geometry">
+    <img src="./images/geometry.svg" alt="geometry">
+  </section>
+
+  <section id="Works">
+    <h2>Projects</h2>
+
+    <div class="project-card-section">
+      <div class="project-card">
+        <img src="./images/project-image.svg" alt="project-image-1">
+        <div class="project-details">
+          <h4>Project name goes here</h4>
+          <div class="project-skill">
+            <ul>
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>Javascript</li>
+            </ul>
+          </div>
+          <button class="project-button">
+            See this project
+            <span class="material-symbols-outlined">
+              arrow_right_alt
+            </span>
```

Comment:
As requested by the previous reviewer, kindly adjust the spacing in your design to match Figma as much as possible. 👍

Your design
![image](https://github.com/donice/Portfolio-Microverse/assets/78251927/878006b4-ea34-4500-bdbc-b0e6565cdd94)


Figma
![image](https://github.com/donice/Portfolio-Microverse/assets/78251927/b7a209fb-aedd-4946-a5a1-d33d8d955798)